Equinox/Level.cpp

```diff
@@ -9,10 +9,21 @@
 #include "ModuleWindow.h"
 #include "IMGUI/imgui.h"
 #include "ModuleEditor.h"
+#include "Quadtree.h"
+#include "ModuleEditorCamera.h"
 
 Level::Level()
 {
 	root = new GameObject;
+
+	vec minPoint = vec(-10000, -100, -10000);
+	vec maxPoint = vec(10000, 100, 10000);
+
+	AABB limits = AABB(minPoint, maxPoint);
+
+	quadtree = new Quadtree(limits);
+
+	//quadtree->Insert(root);
```

Comment:
Could be great to have an explanation why a line is commented. And if that commented line isn't necessary, delete it. ;)